index.html

```diff
@@ -11,123 +11,132 @@
+        <nav class="nav">
+            <img class="nav__icon" src="./images/ic_menu.svg" alt="Menu" title="Menu">
+            <ul class="nav__links">
+                <li><a href="#" class="nav__link"><h2>Home</h2></a></li>
+                <li><a href="#" class="nav__link"><h2>Portfolio</h2></a></li>
+                <li><a href="#" class="nav__link"><h2>About</h2></a></li>
+                <li><a href="#" class="nav__link"><h2>Contact</h2></a></li>
+            </ul>
+        </nav>
     </header>
 
     <!-- Hero section -->
     <section class="hero-container">
-        <h1 class="title">Charlie Giorgania</h1>
-        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.<br>
+        <h1 class="hero__title hero__grid__item title">Charlie Giorgania</h1>
+        <p class="hero__text hero__grid__item paragraph">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.<br>
         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
-        <button type="button" class="bg-dark">
+        <button type="button" class="hero__button hero__grid__item button btn-bg-dark">
             Start Collaboration
         </button>
-
-<!-- Projects section -->
     </section>
         
     <!-- Projects section -->
     <section class="projects-container">
-        <section class="projects-title-container">
-            <h2 class="title">Projects</h2>
-        </section>
         <section class="projects-cards-container">
-            <article class="project-card">
-                <img src="./images/firstProject.svg" alt="Project one" title="Project one">
+            <section class="projects-title-container">
+                <h2 class="projects__title title">Projects</h2>
+            </section>
+            <article class="project-card  bg-dark card1 bg-none">
+                <img class="project__image" src="./images/firstProject.svg" alt="Project one" title="Project one">
                 <div class="projects-text">
                     <h3>Project name goes here</h3>
                     <ul>
                         <li>HTML/CSS</li>
                         <li>Ruby on Rails</li>
                         <li>JavaScript</li>
                     </ul>
-                    <button class="projects-button bg-dark" type="button">
+                    <button class="button projects-button btn-bg-dark" type="button">
                         See this project <img src="./images/Union.svg" alt="Arrow icon">
                     </button>
                 </div>
             </article>
 
-            <article class="project-card">
-                <img src="./images/secondProject.svg" alt="Project two" title="Project two">
+            <article class="project-card bg-dark card2 bg-none">
+                <img class="project__image" src="./images/secondProject.svg" alt="Project two" title="Project two">
```

Comment:
- [ ] As the last reviewer mentioned Please add all project names and technologies and see this project button that appears in the desktop version design and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/160045519-863f20c4-f243-4acc-990e-949230c67efe.png)  | ![image](https://user-images.githubusercontent.com/68796232/160045551-31b21e05-9c14-4c7f-ae1d-7ed1e505a838.png)  |

Reply:
Hi @Omar-Muhamad! Isn't that project name and technologies section a hover feature of each project?

Reply:
I think its fixed and It doesn't work as a hover as well.